core/src/main/scala/com/nulabinc/backlog/migration/common/client/BacklogAPIClientImpl.scala

```diff
@@ -16,19 +18,31 @@ object BacklogAPIClientImpl extends BacklogConfiguration {
   }
 }
 
-class BacklogAPIClientImpl(configure: BacklogConfigure)
-    extends BacklogClientImpl(configure, BacklogAPIClientImpl.client)
+case class IAAH(value: String) extends AnyVal
+
+object IAAH {
+  val empty: IAAH = IAAH("")
+}
+
+class BacklogAPIClientImpl(configure: BacklogConfigure, iaah: IAAH)
```

Comment:
@yuichi0301 Please review this file carefully

Reply:
@shomatan LGTM!